techniques/fileDistribution/copyGitFile/2.3/metadata.xml

```diff
@@ -18,16 +18,34 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   <DESCRIPTION>This technique ensure that some files are the copy of files on the shared folder of the Rudder Root Server, and let you optionally execute a command if the content of the file was modified</DESCRIPTION>
 
   <MULTIINSTANCE>true</MULTIINSTANCE>
+  <POLICYGENERATION>separated</POLICYGENERATION>
   <COMPATIBLE>
     <OS version=">= 4 (Etch)">Debian</OS>
     <OS version=">= 4 (Nahant)">RHEL / CentOS</OS>
     <OS version=">= 10 SP1 (Agama Lizard)">SuSE LES / DES / OpenSuSE</OS>
     <AGENT version=">= 3.2.0">cfengine-community</AGENT>
   </COMPATIBLE>
 
-  <BUNDLES>
-    <NAME>download_from_shared_folder</NAME>
-  </BUNDLES>
+  <AGENT type="dsc">
```

Comment:
Isn't this a bug that should be fixed in previous version of the technique?

Reply:
yes, see https://www.rudder-project.org/redmine/issues/11881

Reply:
i'll do it in a separe issue, if it's ok with you

Reply:
:+1: